src/main/resources/application.yml

```diff
@@ -14,4 +14,14 @@ management:
       jmx:
         exposure.include: "*"
       web:
-        exposure.include: info, health
\ No newline at end of file
+        exposure.include: info, health
+app:
+    pdnd:
+      base-url: "\${PDND_BASE_URL:https://auth.uat.interop.pagopa.it}"
+      config:
+        audience: "\${PDND_ACCESS_TOKEN_AUDIENCE:auth.uat.interop.pagopa.it/client-assertion}"
+        client-id: "\${PAGOPA_PDND_CONFIGURATION_CLIENT_ID:890b7ca9-b402-4dce-9e8d-9a333d22d76d}"
+        kid: "\${PAGOPA_PDND_CONFIGURATION_KID:jxOpPRxM6oFcnnKtICqeW5l7fbxLr45IAsJ8Q9s-fK8}"
+        purpose-id: "\${PAGOPA_PDND_CONFIGURATION_PURPOSE_ID:87520bd5-207a-4616-85d9-10d7bb3e88b8}"
```

Comment:
the purposeId change based on the service to call, it cannot be a global PDND configuration

eg:
app.pdnd.services.anpr.purpose-id